libraries/common/actions/router/historyRedirect.js

```diff
@@ -0,0 +1,22 @@
+import { historyPop, historyReplace } from './index';
+
+/**
+ * Performs a redirect. When the action params contain a location, the current history entry
+ * will be replaced with it. Otherwise the current history entry will be popped.
+ * @param {Object} params The redirect params.
+ * @return {Function} A redux thunk.
+ */
+export function historyRedirect(params = {}) {
+  return (dispatch) => {
+    const { location, state = {} } = params;
```

Comment:
Please don't make the state an empty object. The `history...` actions will pass it through. This means if state is not set, it will be treated as `undefined`. Thas the way to go.

Reply:
done